tensorboard/plugins/hparams/backend_context_test.py

```diff
@@ -897,6 +902,61 @@ def test_experiment_from_data_provider_old_response_type(self):
         """
         self.assertProtoEquals(expected_exp, actual_exp)
 
+    def test_experiment_from_tags_with_hparams_limit_returns_differed_hparams_first(
+        self,
+    ):
+        experiment = """
+            name: 'Test experiment'
+            hparam_infos: [
+              {
+                name: 'batch_size'
+                type: DATA_TYPE_FLOAT64
+                differs: false
+              },
+             {
+              name: 'lr'
+              type: DATA_TYPE_FLOAT64
+              differs: true
+            },
+            {
+              name: 'model_type'
+              type: DATA_TYPE_STRING
+              differs: true
+            }
+            ]
```

Comment:
This closing square bracket should probably not have the same indent as the closing squiggly bracket.